README.md

```diff
@@ -21,7 +21,7 @@ If compared to saving websites with `wget -mpk`, `monolith` embeds all assets as
 ### Options
  - `-i`: Remove images
  - `-j`: Exclude JavaScript
- - `-u`: Specify custom User-Agent
+ - `-a`: Specify custom User-Agent
```

Comment:
I've been thinking that `-u` is more semantic and intuitive, as many users know what User-Agent is, while just Agent may be a bit puzzling. I'm also considering reserving `-a` for something like "don't modify anchors' href links" or "don't include links at all)", or "include audio sources". This way `-v` could be used for video tags. But planning option and flag  names is usually too big of a task to be done with in one sit.